rest/taskana-rest-spring/src/main/java/pro/taskana/RestConfiguration.java

```diff
@@ -57,14 +60,16 @@ public TaskanaEngine getTaskanaEngine(TaskanaEngineConfiguration taskanaEngineCo
 
   @Bean
   @Scope(ConfigurableBeanFactory.SCOPE_SINGLETON)
```

Comment:
All Beans should be created as singletons as a default, not sure if this annotation is really necessary

Reply:
https://docs.spring.io/spring/docs/3.0.0.M3/reference/html/ch04s04.html
You're right. Will be removed